### ISSUE TITLE
Reddit video missing crash

### ISSUE THREAD

**Issue by pmdevita** [pmdevita/GifReversingBot#15]
```
Traceback (most recent call last):
  File "main.py", line 55, in <module>
    result = process_comment(reddit, reddit.comment(message.id), q)
  File "/home/pmdevita/bots/vredditshare/core/process.py", line 120, in process_comment
    if not new_original_gif.analyze():
  File "/home/pmdevita/bots/vredditshare/core/hosts/reddit.py", line 26, in analyze
    url = submission.media['reddit_video']['fallback_url']
TypeError: 'NoneType' object is not subscriptable
```

**Comment by pmdevita** [comment]
Might have been already fixed, this is from vredditshare

**Comment by pmdevita** [comment]
Fixed in 56a69a1dd275c83b64628913a1c83b79386efb77
